src/scss/_settings.scss

```diff
@@ -29,8 +29,8 @@ $button-font: "Open Sans Condensed", sans-serif;
  * GLOBAL SETTINGS
  */
 $breakpoints: (
-  desktop-bp: 1300px,
-  smdesktop-bp: 1024px,
-  tablet-bp: 768px,
-  mobile-bp: 450px
+  wide-bp: 1300px,
```

Comment:
I think we can drop the `-bp` on these, it seems unnecessary and just additional typing with no value.